tpl/footer.php

```diff
@@ -7,7 +7,7 @@
 The text you paste here will be reachable by a unique URL that is long enough to prevent brute-forcing. Also, these identifiers are randomly chosen. You post will remain on the server for at most 30 days.
 </p>
 <p>
-<?php echo $_SERVER['SERVER_NAME']; ?> and the author are not responsible for any content that is pasted here, nor the confidentiality of your data. Reliability of this service is not a right.
+<?php echo $config['server_name']; ?> and the author are not responsible for any content that is pasted here, nor the confidentiality of your data. Reliability of this service is not a right.
```

Comment:
Config server_name is set in de config.php file, and is used in an another template